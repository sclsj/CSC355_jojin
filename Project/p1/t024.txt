record this_is_an_error_period_is_not_a_legal_double
{
  a = .;
}
