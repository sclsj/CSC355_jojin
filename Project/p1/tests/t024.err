Syntax error on line 3 '.' is not a legal token.
Parse error on line 3 reported by parser: syntax error, unexpected error, expecting int constant or string constant or double constant or month.
