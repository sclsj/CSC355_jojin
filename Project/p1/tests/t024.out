parser.cpp::main() reading file tests/t024.in
parser.cpp::main() about to call yyparse()
type zero or more record declarations followed by ^D:

------------------------------------------------------
record this_is_an_error_period_is_not_a_legal_double
{

------------------------------------------------------

parser.cpp::main() after call to yyparse()

yyparse() found errors

parser.cpp::main() end of program.
